modules/kmm-building-and-signing-a-moduleloader-container-image.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-building-and-signing-a-moduleloader-container-image_{context}"]
+= Building and signing a ModuleLoader container image
+
+Use this procedure if you have source code and need to build your image first.
+
+The YAML file shown below should build a new container image using the source code from the repository. The image produced is saved back in the registry with a temporary name, and this temporary image is
+then signed using the parameters in the `sign` section.
+
+The temporary image name is based on the final image name and is set to be `<containerImage>:<tag>-<namespace>_<module name>_kmm_unsigned`.
+
+For example, using the YAML below Kernel Module Management (KMM) builds an image named `quay.io/chrisp262/minimal-driver:final-default_example-module_kmm_unsigned` containing the build with unsigned kmods and push it to the registry. Then it creates a second image named `quay.io/chrisp262/minimal-driver:final` that contains the signed kmods.
```

Comment:
Please rename `quay.io/chrisp262` with `example.org/repository` everywhere in this file.

Reply:
changed.